AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSConstants.h

```diff
@@ -0,0 +1,10 @@
+#import <Foundation/Foundation.h>
+
+static NSString *const kMSTokens = @"tokens";
```

Comment:
It doesn't seem all those constants are shared by multiple classes. Please move this to `MSTokenResult.h`

Reply:
It is shared in 3 places. Hence the split to constants